app/src/main/kotlin/com/simplemobiletools/dialer/activities/ConferenceActivity.kt

```diff
@@ -4,22 +4,24 @@ import android.os.Bundle
 import com.simplemobiletools.commons.helpers.NavigationIcon
 import com.simplemobiletools.dialer.R
 import com.simplemobiletools.dialer.adapters.ConferenceCallsAdapter
+import com.simplemobiletools.dialer.databinding.ActivityConferenceBinding
 import com.simplemobiletools.dialer.helpers.CallManager
-import kotlinx.android.synthetic.main.activity_conference.*
 
 class ConferenceActivity : SimpleActivity() {
 
+    private lateinit var conferenceActivityBinding : ActivityConferenceBinding
```

Comment:
https://github.com/SimpleMobileTools/Simple-Commons/blob/0bdbc422dd71e67e0264b01e72799448cc5c23f2/commons/src/main/kotlin/com/simplemobiletools/commons/activities/LicenseActivity.kt#L17

Reply:
updated

Reply:
it would be great to update it everywhere, not just this place

Reply:
updated!